gcloud/vision/test_client.py

```diff
@@ -0,0 +1,119 @@
pXLw5DQCV7EiBFQ2+/zdNJU/Mk6/56uJKussE8/SEvDFV87yRTrSX72Q/5tqdpS8846PgXtKL7rSNupD9rOxMfT9hfvTv7yZ/5u4Z/HKhxtOvbDg1fuP9IAA/hzHO12ROX/v6Vz4NlbwtamEXN5ZAkydS1SFHkPurgrWaAlCTWsbDGfSpIf0nI9BbBa2nh1GNzB/TY7EyznkMWg3aybULLsNGw5E8+75Mu74r962QeOfzyKOLYm5v4FTn0e0r8FwCxAOjNDO46pfujmzn3P3RItb6rLpZElSsGb7ECg/FygYm4kL0eVQdejDLKuxHtm5V7fZxLG8Sup2hV+Xz6ar0gBulIwY6izfJHTlTj/ueuy/G7b+bCZmmQhbX1lV/KeF//18X8IQ+anpth3oHlm/u4jAnBhdHlPO4Tqf/92Z+8VG8LjmxpySTWg6jxOjJh+RWEObqm+ejAyJCtLla4SxNXQolfztyf7rHLioEgkpDlBPc76X2mSp/5ih7Y9rpm6sO1saz5d/PzO+LqXf/D4J0KYTzx7m02aZxuPsw5wfsUe2iFU/sdtnQMXztj7N9fNRY1QxmKnXkSkD67k5UCaybWW/MgyzUcfRI3KCmpUTlMln+r1U4FbGFEpWUQsBY09xJqHIFPSpdRX2t7Oz6UHP3Fb550//+GFfwlhLoG9wPLDAcTDBTBAvAaaCYT/cGf3SC3wO88ft1unK6xLUsX7AtFCenMOcOFW+RViviqr2y/1JKCvpMJ1BY0wVL47eApOyfUpgE0LlZwFLiRRXJIFMHwnxTWdatNJreNk/6Hkjvd8ZfnP3viJ5s0RHJ2C+5oPE7gPN8A0s/5uzdGI4HO70qV7jqW3XLrWNiatXBAaxClOFCPDyQXtk9BkmKimQ8lyGdorRR7CcXMy2CxFV2BlDBhQecCid8BqSSV3MkaGazkXtJ1xC47vPZB8+o2fWHrH+7/R3T0acbTt2NeE1sOJwcMKcH4lsWNpbQXz7YPOf+z2+LZL19kjayIuGA1opA68qkrRsc0Xp31qL1gg5bwpQx3mh/fqYXfnlGUrsrJvS6kfpw5JssuavRXgaqo9dZyFHT2unaq2nFQ7zhw/lh796q74vVf/7eKHb92fHllb4fB8zD548BZIjxWAAVzTsTQyQjy/rNGHb+3sCoTbt42aybrI5kCzDsHqc8srjwTJcA8UFYo2A6oyWGN2gs80BJhf4eeBv9dBxmNPHec/p/m+mseTC9dH82aiGmfBC99y6pse23LiFh37DyVfue7LrXe89qNLX1ru6vLICPuOdzgIp5/6ezQDDOCThOYkLNuQyhfuSRY/fkf8zQun7exMxMa6MCEeSZ36vEWFqNes050f6M3YM276LT50sEsEJ7GW/Qo/D51w1yO/OZBiby3UcapZo580A5Ouz8KOXY9rO3WtVG3LG9tMZf5Ysu9L34v/5mc/tPi3n9wZ76+HLDQ8e5YS5ngEz0F/JAGGTCd1E8/iRBV/eFnN33+7s3t2yd+8edy4huiGhsiIT1WcU59nXwTNgFYn+fGAuRSXUm/iCr649g7WlnLgoWwY9cKH+QLpHcadvSZFm1+Xqd9eu9/Cl+3m4HYc2vbqmk5N05lKx8viXDK/80Dyyf/yz82/+v1PLX9zoaOLE1UOLsXc3+Hsu0FnFKp/hC4zPs64a3POcswEUP2DFzYu/NmnVV6yaco+Z2I8GE8CQSNBK4GaSJDQiATSq70Vm1XSizX9cy5ykoHmAZUBzpSWykZK7a4yA0AGuuSqz9J5kpVt9sDOLWX1sUdiJxIrYaocX0gX9s+5L//9rd1P/dfPL98NdBoRx22NQwsLLDySUvtoAbi4opERppKE9UnCCFD7gx+tPfHfXzLywq0zwTNrI3JOpW5JreAj4zOulxGTg907pNpk7RTECoIpUXRLQ5XhUOOgO6Rk6lh6Ry9mjdw0VbzzSuJVEzCxM4GDbtPRbumh+46mX//o7a3P/9fPtb8HtMOQVhhysNVijrMYdnysAlxcI+snmTk4zzQwCgSvfubI1qsvja560vroqrERLmg07IiEhiQAHxiVvOxUglyyc96X2oxtpaWWD8NH62iJICd5K0nx/T1XM79W1Xk0BZN6CfNU3/Kyay222PXdg/FNO74T3/T+r7fuy5X80vpJjh2c5ygPs/vzWAQYQNbBSFJjcq7NDNAAuGxjtPaXrojOf855lcvOGTOXjo3I1rBqRiqRwYeCDwQ1glpRJFfhGXdLxEg/czUkwUVrBHzW+Esd4D3iVMQrJlVMonRjT9LxrcWW3ndo0X/ny7u73/7QzfE93z4Qz+afuDxV42jYZv5wBqw+aiaUR+clQHVkhAlJmG4m1PNdtPGsbXbspRdVN16xubJ984Q8abJhtoSRmQgDGYsqYjTvYKsmy26AqJqsF2C5hg9FxIuAivHaO59KnCfuqk9SXUxif3x+2e/dd1y/e/O+7s5P3tU58LU9bjEPK/p6SFNDjrZaLOQ+rT4aJ/LRfoXj4zSqMDG3wGgCVbI7AnjpReH0c7dVz902Y9dvnpat0w27YSQyU9VQJwNrxquRRGFQ6g1CPxKVpEon1jh1fqGTyHwr9nPHlt0D+47pfXuOuoNf2tO5/5N3JQXhLQY6IXSmxlnqwPGFBZYfKX/2+xngntW9BaL5MUZsTL3ZYTSGkRxsITs0fOQFF1RnzpuWyXMadmxqRBqjNVMdCaRaCQkDm/FXU6e+m5C0Uu0stX1nrqXLh5bd4u5jOv+FXZ2jrZhWyTvuRNCqV1lyEc3JRVp7M7D9Y2HSHksADz93AISjUEtr1AJHbSmmSv9U7JVofA9WilZ4yMloRCe1tIM27SVo088T6WNxor5frmAThNUpQmIq7YhK4AhcRBB4rCrGa5id8yKJF8GnBmdj0tSS1mK6RHQ7cyT7+4A+5q/vJ4BXGttJeLAnRKL1sSidj1+PX49fj1+PX49fj1+P3ut/A6a1MamkZwQCAAAAAElFTkSuQmCC"
```

Comment:
1. You have `IMAGE_CONTENT` twice
1. Do you really need a real image for a unit test?
1. If you actually do, put it in a file.